x/incentives/keeper/hooks.go

```diff
@@ -31,6 +33,7 @@ func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochIdentifier string, epochNumb
 		// or non-perpetual and for synthetic denoms.
 		// We distribute to perpetual synthetic denoms elsewhere in superfluid.
 		// TODO: This method of doing is a bit of hack, should clean this up later.
+		// TODO: Discuss/Create issue for the above.
 		distrGauges := []types.Gauge{}
 		for _, gauge := range gauges {
 			isSynthetic := lockuptypes.IsSyntheticDenom(gauge.DistributeTo.Denom)
```

Comment:
Should I create an issue to clean up this hack or is it acceptable?

Reply:
I think an issue + removing comment makes sense

Reply:
https://github.com/osmosis-labs/osmosis/issues/2008